convention.md

```diff
@@ -74,10 +74,12 @@ As a consequence a new MQTT connection to the brocker is required per published
 
 ## Base Topic
 
-The MQTT base topic you will see in the following convention will be `homie/`.
-If this base topic does not suit your needs (in case of, e.g., a public broker), you can choose another.
+The root topic in this document is `homie/`.
+If this root topic does not suit your needs (in case of, e.g., a public broker or because of branding),
+you can choose another.
 
-Be aware, that only the default base topic `homie/` is eligible for automatic discovery by third party controllers.
+Homie controllers are required to perform auto-discovery on the wildcard topic "+/+/$homie"
+to stay independant of the choosen root topic.
```

Comment:
Why not make it even more open, `{base_topic}/+/$homie`

Cant 'homie/location1/' also be a base topic ? The controller would only need to know a list of base topics to search ?

Reply:
default would be 'homie/'

Reply:
The problem is that you can't subscribe to `#/$homie`, because # is only supported as the last character.

Reply:
so the default would be `+/+/$homie` but i still think controller should allow you to add your own base paths to search on ?

Id recommend what @davidgraeff has said but not limit to only this ? That's all im trying to say??

Reply:
Controllers are free to show all or just a selection of discovered homie devices. That implies that they can limit the subscription to a specific root topic. Why should this be extra mentioned? The document should be as concise as possible, I think.